feed-creator.js

```diff
@@ -33,8 +34,8 @@ module.exports = function (meta, data) {
 
       feed.addItem({
         title: item.title,
-        id: `https://bkardell.com/bcd-watch/${item.file}`,
-        link: `https://bkardell.com/bcd-watch/${item.file}`,
+        id: `https://bcd-watch.igalia.com/${item.file}`,
+        link: `https://bcd-watch.igalia.com/${item.file}`,
```

Comment:
```suggestion
        link: `https://bcd-watch.igalia.com/${data.path}/${item.file}`,
```

Reply:
`${data.path}` should be `${directory}` (see lines 19-21).